### ISSUE TITLE
Tile content is hidden when it is used as ExpandableBlock child

### ISSUE THREAD

**Issue by irenajur** [iTwin/iTwinUI#123]
#### Environment
- __Package version(s)__: 1.4.1
- __Operating System__: Windows 10
- __Browser name and version__: Opera 76.0.4017.154

#### Code Sandbox
Link to a minimal repro (fork [this code sandbox](https://codesandbox.io/s/itwinui-react-example-son74)): https://codesandbox.io/s/itwinui-react-example-forked-09ru0?file=/src/App.tsx

#### Steps to reproduce
1. Add Tile as ExpandableBlock child component
1. Expand Block

#### Actual behavior
Tile name, description, metadata, options & etc. are not visible.

#### Expected behavior
All Tile component content is visible if defined.

#### Additional information
Locally managed to workaround this issue by unsetting following styles: 
```css 
  .iui-expanded {
    .iui-content {
      transform: unset !important;
      height: unset;
      opacity: unset;
    }
  }
```

**Comment by mayank99** [comment]
Thanks for the issue. We have renamed the class to `.iui-expandable-content` in #124.

This will be released in itwinui-css v0.18.0 and should be available in the next itwinui-react release (1.6.1 or 1.7.0) without any unsets needed from your side.

**Comment by mayank99** [comment]
Released in [itwinui-react 1.6.1](https://github.com/iTwin/iTwinUI-react/releases/tag/v1.6.1) 🎉
